collections/_infographics/emissions/fossil-fuels-emissions.md

```diff
@@ -1,16 +1,60 @@
+
+### Gas
+
+* **Usage:** Electricity and heat production in households (heating, cooking, water heating) and industry.
+* **Extraction and transportation:** In total, [4000 billion m<sup>3</sup>](https://www.bp.com/content/dam/bp/business-sites/en/global/corporate/pdfs/energy-economics/statistical-review/bp-stats-review-2019-natural-gas.pdf) were extracted in 2019, i.e. about 3.2 billion tons of natural gas. The largest producers include the USA (21% of world production), Russia (17%) and the Gulf States (together 18%). [Transportation of natural gas](https://en.wikipedia.org/wiki/Natural_gas#/media/File:Global_Gas_trade_both_LNG_and_Pipeline.png) occurs through pipelines and ships carrying liquefied gas.
+* Burning natural gas is significantly more efficient than burning coal for generating electricity. <glossary id="ipcc">IPCC</glossary> estimates the emission intensity of natural gas in the range of 410-650 g CO<sub>2</sub>eq/kWh (including extraction and transport); for coal combustion, it indicates a range of 740-910 g CO<sub>2</sub>eq/kWh. This fact represents an important reason for the so-called <glossary id="phase-out">coal phase-out</glossary>, which occurs when a combination of renewable energy sources and gas-fired power plants replaces coal-fired power plants.
+* In addition to CO<sub>2</sub> emissions from combustion, natural gas extraction and transport also produces methane emissions. These are mainly [gas leaks during mining](https://en.wikipedia.org/wiki/Fugitive_emission) estimated at around [100 million tons of CH<sub>4</sub>](https://www.carbonbrief.org/scientists-concerned-by-record-high-global-methane-emissions) per year, equivalent to about a third of anthropogenic methane emissions.
+
+### Cement
+{% comment %}
+{% include preview-box.html
+    title="More details on emissions from cement production"
+    text="The production of one tonne of cement creates approximately 1 to 1.2 tonnes of CO₂. Our explainer describes this process in more detail."
+    slug="emise-vyroba-cementu"
+%}
+{% endcomment %}
+
+* Heating a mixture of limestone and clay to temperatures of around 1450 °C produces cement. Additionally, the heat causes the limestone to decompose into calcium oxide and carbon dioxide. Approximately half of the total CO<sub>2</sub> emissions from cement production arise from the CaCO<sub>3</sub> + heat ⟶ CaO + CO<sub>2</sub> chemical reaction. The other half of the total emissions relates to coal or gas combustion. In the category *cement*, only emissions from the chemical reaction are entered in the graph. The graph includes combustion emissions for the respective fossil fuels.
+* Total emissions from cement production (including heating fuels) are approximately 2.8 billion tons of CO<sub>2</sub>, or about 8% of world CO<sub>2</sub> production. If the cement industry were a state, its emissions would be the third largest in the world, after China and the USA.
```

Comment:
The total emissions from cement production (including heating fuels) are approximately 2.8 billion tons of CO<sub>2</sub>, or about 8% of world CO<sub>2</sub> production. If the cement industry were a state, its emissions would be the third largest in the world, after China and the USA.

Reply:
I updated the text but had a question here with regards to the word `"state"`.
In the sentence: "If the cement industry were a `state`, its ...".

Should we be using `state or this be `country`?

I noticed deepl.com sometimes using the word `state` and other times using `country`. Google translate does the same thing. Note: For a nation or territory considered as an organized political community under one government such as the "United States of America" and the "European Union" the word "state" or "member states" this makes sense.

Reply:
I do agree with Willem that "country" is a better term here. "State" is probably also a residuum of the Czech  language (we use the direct translations of "state" and "country" almost interchangeably).